### ISSUE TITLE
Under latest stackage nightly, test suite can no longer find fixture files

### ISSUE THREAD

**Issue by kvanbere** [hspec/hspec#422]
This has me scratching my head.

The repo is here:
https://github.com/onrock-eng/github-webhooks

Notice the difference in output vs nightly with only an upgrade in resolver:

Pass:
https://travis-ci.org/onrock-eng/github-webhooks/jobs/599099622

Fail:
https://travis-ci.org/onrock-eng/github-webhooks/jobs/599099625

**Comment by sol** [comment]
@kvanbere the build logs are not available anymore.  I'm not sure what happened their.  If you want me to look into this I'll need a way to reproduce it.

Closing this for now, but please feel free to open a new issue.

**Comment by kvanbere** [comment]
Hi @sol,

Yeah sorry, we rebranded so the repo now lives under @cuedo.

Now.. talk about reaching right back into my memory.. I think this was caused by an upgraded dependency of the 'time' package, which changed the way that UTC strings were decoded in their default 'read' implementation. Even though hspec seemed to be barfing the error, it wasn't the root cause. I don't think there's any action items for hspec here, sorry.

Cheers mate
